pkg/cmd/server/api/types.go

```diff
@@ -54,12 +54,12 @@ var (
 	KubeAPIGroupsToAllowedVersions = map[string][]string{
 		APIGroupKube:           {"v1"},
 		APIGroupExtensions:     {"v1beta1"},
-		APIGroupApps:           {"v1alpha1"},
+		APIGroupApps:           {"v1beta1"},
 		APIGroupAuthentication: {"v1beta1"},
 		APIGroupAutoscaling:    {"v1"},
 		APIGroupBatch:          {"v1", "v2alpha1"},
 		APIGroupCertificates:   {"v1alpha1"},
-		APIGroupPolicy:         {"v1alpha1"},
+		APIGroupPolicy:         {"v1beta1"},
```

Comment:
It wasn't fully removed, but the only files in that GV are register.go and types.go. There are no generated files there. I'm wondering if someone forgot to fully remove it?